bankbarcode/bankbarcode.py

```diff
@@ -38,16 +38,21 @@ def code(self):
         """
         raise NotImplementedError('This method is not implemented!')
 
-    def save(self, path):
+    def save(self, path, writer_options=None):
         """
         Save barcode in SVG format.
 
         :param path: path to SVG file with or without ".svg" extension
+        :param writer_options: Common options from pyBarcode http://pythonhosted.org/pyBarcode/writers/index.html?#common-options
         :return: a string with the name of the file generated
         """
         path = self._strip_dotsvg(path)
 
-        writer_options = {'font_size': 6}
```

Comment:
I think it would be better if we have a default options:

```python
defaults = {
  'font_size': 6
}

if writer_options is None:
    writer_options = defaults.copy()
else:
   for k, v in defaults.items():
       writer_options.setdefault(k, v)
```